dependencies.yaml

```diff
@@ -225,7 +225,9 @@ dependencies:
           - &gmock gmock>=1.13.0
           # Hard pin the patch version used during the build. This must be kept
           # in sync with the version pinned in get_arrow.cmake.
-          - libarrow==12.0.1.*
+          # Temporarily building a specific build string in order to choose one
+          # that is compatible with the aws-sdk-cpp version required above
+          - libarrow==12.0.1.*=hb6645e0_10_cpu
```

Comment:
Do we need to make this pinning architecture-specific? I think we’ll need a build pinning for x86-64 and aarch64. Shouldn’t need to worry about Python or CUDA versions for a C++ CPU build.

Reply:
Done.